src/router/index.js

```diff
@@ -151,9 +151,9 @@ const routes = [
           {
             path: "",
             name: "settings",
-            component: Settings
-          }
-        ]
+            component: Settings,
+          },
+        ],
```

Comment:
Umbrel hasn't run their configured linter in over two years now. Even though I understand that you want to keep the code style, maybe try to avoid putting the changes in this PR.

Reply:
Sounds good, removed that commit.